tests/test_utils.py

```diff
@@ -223,6 +223,22 @@ def test_ipv4_network(self):
         self.assertTrue(a1 in n3)
         self.assertFalse(a1 in n4)
 
+    def test_ipv4_list(self):
+        n1 = utils.IPv4Network(u"10.0.0.0/16")
+        n2 = utils.IPv4Network(u"10.0.1.0/24")
+        n3 = utils.IPv4Network(u"10.0.0.5/32")
+        n4 = utils.IPv4Network(u"192.168.1.0/24")
+        IPV4_list = utils.IPv4List([n1, n2])
```

Comment:
I think it is a bit weird that n2 and n1 overlap. Testing is never going to be checking the second value as if it matches the first, it won't check the second.